lib/aerosol/aws_model.rb

```diff
@@ -76,6 +77,7 @@ def from_hash(hash)
 
       instance = new!
       instance.from_aws = true
+      instance.launch_template = hash[:launch_template][:launch_template_name] if hash[:launch_template]
```

Comment:
I'm curious why this line is necessary? Feels very bizarre to have this in the `aws_model` class since not everything is a launch template or ASG.

Reply:
I originally thought about overriding `from_hash` in the subclasses. Is that better?

```def from_hash(hash)
instance = super
instance.launch_template = hash[:launch_template][:launch_template_name] if hash[:launch_template]
```
in Instance and AutoScaling?

Reply:
I think so, as long as it doesn't conflict with the latter portions of the `#from_hash` method.

That aside, why do we set the value to be the name? I forget exactly how this gem works, but wouldn't it want to be the actual AwsModel representing the launch template?

Reply:
This was to allow easy configuration migration so that I could just do `launch_configuration` -> `launch_template` in the `aerosol.rb` files instead of having to edit them more than that.